__tests__/_extend-jest.ts

```diff
@@ -0,0 +1,41 @@
+import { FetchMock } from './_helper'
+
+export function extendExpect() {
+  expect.extend({ toHaveBeenCalledFor, toHaveBeenCalledOnceFor })
+}
+
+function toHaveBeenCalledFor(
```

Comment:
grammar
```suggestion
function toHaveBeenCalledWith(
```

Reply:
And possibly be explicit here (since this is for fetch anyways):
```suggestion
function toHaveRequestsTo(
```